kernel/univ.mli

```diff
@@ -319,15 +319,24 @@ module AUContext :
 sig 
   type t
 
+  val repr : t -> UContext.t
+  (** [repr ctx] is [(Var(0), ... Var(n-1) |= cstr] where [n] is the length of
+      the context and [cstr] the abstracted constraints. *)
+
   val empty : t
+  val is_empty : t -> bool
 
+  (** Don't use. *)
   val instance : t -> Instance.t
-                        
+
```

Comment:
But then for printing we should be able to reuse the idea of having functions parameterized by a (Level.t -> stdppcmds) function that would resolve de Bruijns to user names, shouldn't it?